WalletConnect/web3inbox
# Experiment/sync chat

# Changes
- Sync `chatSentInvites` and `chatThreads` by listening to their store updates
- Hide `chatSentInvite` that already has a thread, to account for the 1-2 second delay of sync updates causing both the thread and sent invite to show up
- Conform to new spec for chat client since these changes required a package bump

# How was this tested
- Open web3inbox on brave and firefox, 
- send invites and observe the fact they're syncing on the invite being sent and accepted. 
- Send invites and observe that accepting them does not cause discrepancies. 

# Notes
These changes do add a small delay for invites etc, looking into how we can speed these up by possibly not waiting for the `syncSet` request to succeed before resolving the `set` function
